RSVflights/src/screens/MyFlights.js

```diff
@@ -23,21 +29,52 @@ export default function MyFlights(props) {
   }
   return (
     <>
-      <StatusBar barStyle="light-content" />
+      <StatusBar />
       <SafeAreaView style={styles.background}>
         {user ? (
           <View>
-            <Text style={styles.title}>My Flights</Text> 
-            <Icon 
+            <Text>Logeado</Text>
```

Comment:
This is necessary?

Reply:
It was for test, but not really is important.